code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -393,4 +393,11 @@
 /obj/machinery/atmospherics/components/unary/cryo_cell/can_see_pipes()
 	return 0 //you can't see the pipe network when inside a cryo cell.
 
+/obj/machinery/atmospherics/components/unary/cryo_cell/return_temperature()
+	var/datum/gas_mixture/G = AIR1

```

Comment:
Why aren't you using AIR1 instead of G?

Reply:
this was copy pasted from the original hardcoded cryo return temp code

Reply:
it's supposed to be slightly faster to get a local var reference to the global